bundle/Resources/views/default/content/full.html.twig

```diff
@@ -1,12 +1,86 @@
+            content: "click to copy";
+            display:none;
+            white-space: nowrap;
+            background-color: whitesmoke;
+            font-family: sans-serif;
+            font-size: 13px;
+        }
+        code:not(.empty):active::after {content: "copied"}
+        code:not(.empty):hover::after{position: absolute; left: calc(100% + 10px); display: inline}
+    </style>
+    <script type='application/javascript'>
+        function copyText(element) {
+            var aux = document.createElement('input');
+            aux.setAttribute('value', element.innerHTML);
+            document.body.appendChild(aux);
+            aux.select();
+            document.execCommand('copy');
+            document.body.removeChild(aux);
+        }
+    </script>
+
+    <h1>{{ content.name }}</h1>
+
+    <dl>
+        <dt>ID</dt>
+        <dd><code onclick="copyText(this)">{{ content.id }}</code></dd>
+        <dt>Location ID</dt>
+        <dd>
+            {% if location is defined %}
+                <code onclick="copyText(this)">{{ location.id }}</code>
+            {% else %}
+                <code class="empty">[empty]</code>
+            {% endif %}
+        </dd>
+        <dt>Current version</dt>
+        <dd><code onclick="copyText(this)">{{ content.innerVersionInfo.versionNo }}</code></dd>
+        <dt>Content type</dt>
+        <dd><code onclick="copyText(this)">{{ content.contentInfo.contentTypeIdentifier }}</code></dd>
+    </dl>
+
+    <h3>Fields</h3>
+
     {% for identifier, field in content.fields %}
-        <h3>{{ field.name }} [{{ identifier }}/{{ field.fieldTypeIdentifier }}]</h3>
-        {% if not field.isEmpty() %}
-            {{ ng_render_field(field) }}
-        {% else %}
-            <p>Field is empty.</p>
-        {% endif %}
+        <details>
+            <summary>{{ loop.index }}. {{ field.name }}</summary>
+            <blockquote>
+                <dl>
+                    <dt>ID</dt>
+                    <dd><code onclick="copyText(this)">{{ field.id }}</code></dd>
+                    <dt>Field definition ID</dt>
+                    <dd><code onclick="copyText(this)">{{ field.innerFieldDefinition.id }}</code></dd>
+                    <dt>Identifier</dt>
+                    <dd><code onclick="copyText(this)">{{ identifier }}</code></dd>
+                    <dt>Field type identifier</dt>
+                    <dd><code onclick="copyText(this)">{{ field.fieldTypeIdentifier }}</code></dd>
+                    <dt>Description</dt>
+                    <dd>
+                        {% if field.description is not empty %}
+                            {{ field.description|default('<code>[empty]</code>') }}
+                        {% else %}
+                            <code class="empty">[empty]</code>
+                        {% endif %}
+                    </dd>
+                    <dt>Contents</dt>
+                    <dd>
+                        {% if not field.isEmpty() %}
```

Comment:
Maybe write this `{% if not field.empty %}` to make it a bit leaner?

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/130/commits/4823ad7ee41d4a3499f5bdb5354956e0cb052bfb